x/delayedack/eibc.go

```diff
@@ -74,7 +74,7 @@ func (im IBCMiddleware) eIBCDemandOrderHandler(ctx sdk.Context, rollappPacket co
 // It validates the fungible token packet data, extracts the fee from the memo,
 // calculates the demand order price, and creates a new demand order.
 // It returns the created demand order or an error if there is any.
-func (im IBCMiddleware) createDemandOrderFromIBCPacket(fungibleTokenPacketData transfertypes.FungibleTokenPacketData,
+func (im IBCMiddleware) createDemandOrderFromIBCPacket(ctx sdk.Context, fungibleTokenPacketData transfertypes.FungibleTokenPacketData,
```

Comment:
it would be really nice to cover this file with unit tests, especially since it has some critical features, and now we are adding changes without the possibility of regression testing

Reply:
for example, scenarios such as `eibc_fee < bridging_fee`...

Reply:
added #916